src/tests/ftest/io/parallel_io.py

```diff
@@ -71,7 +71,7 @@ def statvfs_pool(self, path):
         statvfs_list = []
         for _, pool in enumerate(self.pool):
             dfuse_pool_dir = str(path + "/" + pool.uuid)
-            statvfs_info = self.stat_bfree(dfuse_pool_dir)
+            statvfs_info = self.stat_free_blocks(dfuse_pool_dir)
```

Comment:
From the looks of it.
```suggestion
            statvfs_info = self._stat_free_blocks(dfuse_pool_dir)
```